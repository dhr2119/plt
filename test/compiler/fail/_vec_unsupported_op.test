int main() {
  vector int [2] v;
  float i;
  i = 5.0;
  v = | 1 | 2 |;
  v = v / i;
  return 0;
}
