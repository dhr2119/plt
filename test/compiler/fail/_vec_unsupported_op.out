Fatal error: exception Exceptions.InvalidBinopExpression("Arithmetic operators on unsupported type")
